fdbclient/include/fdbclient/Tenant.h

```diff
@@ -37,6 +37,10 @@ Key idToPrefix(int64_t id);
 int64_t prefixToId(KeyRef prefix, EnforceValidTenantId = EnforceValidTenantId::True);
 
 constexpr static int PREFIX_SIZE = sizeof(int64_t);
+
+// This system keys stores the tenant id prefix that is used during metacluster/standalone cluster creation. If the key
+// is not present then we will assume the prefix to be 0
+const KeyRef tenantIdPrefixKey = "\xff/tenant_id_prefix"_sr;
```

Comment:
To make this slightly more convenient to use, we could define this here as:

```
KeyBackedProperty<int64_t>& tenantIdPrefix();
```

And implement it in Tenant.cpp like the following (we do something similar for the metacluster key-backed types in Metacluster.h):

```
KeyBackedProperty<int64_t>& tenantIdPrefix() {
	static KeyBackedProperty<int64_t> instance(TenantMetadata::instance().subspace.withSuffix("idPrefix"_sr));
	return instance;
}
```

Then users could write:

```
wait(TenantAPI::tenantIdPrefix().get(tr));
```